src/compressed_tensors/compressors/quantized_compressors/pack_quantized.py

```diff
@@ -133,14 +159,29 @@ def decompress_weight(
         original_shape = torch.Size(compressed_data["weight_shape"])
         num_bits = quantization_args.num_bits
         unpacked = unpack_from_int32(weight, num_bits, original_shape)
+
+        # NOTE: this will fail decompression as we don't currently handle packed zp on decompression
```

Comment:
maybe i could take a stab at this in a future PR. looks like some of the code is there, and users will probably want this?

Reply:
We'll want it to be able to run CompressedLinear and asym lm-evals with hf. 
And sure - I can walk you through the decompression changes.